File: Python/aprendizaje/Ejercicios/Tablas.py
"""
6. Mostrar todas las tablas de multiplicar del 1 al 10.
"""
for i in range(1,11):
    print(f"Tabla del {i}")
    print("-----------------")
    for j in range(1,11):
        print(f"{i} x {j} = {i*j}")
    print("-----------------")

File: Rust/basicos/Tablas.rs
// 6. Mostrar todas las tablas de multiplicar del 1 al 10.
fn main(){
    println!("Tablas de multiplicar del 1 al 10");
    println!("##################################");
    for i in 1..=10 {
    println!("Tabla del {}", i);
    println!("----------------------------------");
        for j in 1..=10 {
            println!{"{} * {} = {}", i, j, (i*j)};
        }
        println!("##################################");
    }
}
